MariaOlivenbaum/HW5/main.py

```diff
@@ -0,0 +1,52 @@
+print("***THE FIRST TASK***")
+
+list_size = int(input("Enter the number of numbers in the list: "))
+
+list_of_integers = []
+for i in range(list_size):
+    list_of_integers.append(int(input(f"Enter {list_size-i} more numbers for the list: ")))
+
+list_of_floats = [float(i) for i in list_of_integers]
+
+print(f"Our list but in float type: {list_of_floats}")
+
+# ==============================
+
+print("\n***THE SECOND TASK***")
+
+last_number_fibonacci = int(input("Enter the last number in this fibonacci sequence: "))
+
+fibonacci_sequence = [0, 1]
+
+if last_number_fibonacci == 0:
+    print(f"The Fibonacci_sequence till {last_number_fibonacci}:", fibonacci_sequence[0])
+else:
+    while (fibonacci_sequence[-1] + fibonacci_sequence[-2]) <= last_number_fibonacci:
+        fibonacci_sequence.append(fibonacci_sequence[-1] + fibonacci_sequence[-2])
+    print(f"The Fibonacci_sequence till {last_number_fibonacci}:", fibonacci_sequence)
+
+# or we can use FOR
+# else:
+#     for i in range (last_number_fibonacci):
+#         if (fibonacci_sequence[-1] + fibonacci_sequence[-2]) < last_number_fibonacci:
+#             fibonacci_sequence.append(fibonacci_sequence[-1] + fibonacci_sequence[-2])
+#         else:
+#             break
+#     print(f"The Fibonacci_sequence till {last_number_fibonacci}:", fibonacci_sequence)
+
+# ==============================
+
+print("\n***THE THIRD TASK*** ")
+
```

Comment:
good job
look at this implementation:
input_number = int(input("Input natural number: "))

fact = 1

if input_number == 1 or input_number == 0:
    print("Factorial {}! =".format(input_number), fact)
elif input_number < 0:
    print("Factorial non positive number don't exist")
else:
    for item in range(1,input_number + 1):
        fact *= item
    print("Factorial {}! =".format(input_number), fact)